.github/PULL_REQUEST_TEMPLATE/feat-fix-refactor.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Feature / Fix / Refactor
+about: Adds a feature, fixes a bug, refactors code; changes production code.
+---
+
+## Description
+
+<!-- Add a description of the changes that this PR introduces and the files that
+are the most critical to review. -->
+
+closes: #XXXX
+
+---
+
+### Author Checklist
+
+*All items are required. Please add a note to the item if the item is not applicable and
+please add links to any relevant follow up issues.*
+
+I have...
+
+- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
+- [ ] added `!` to the type prefix if API or client breaking change
+- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
+- [ ] provided a link to the relevant issue or specification
+- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
+- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
+- [ ] added a changelog entry to `CHANGELOG.md`
+- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
+- [ ] updated the relevant documentation or specification
+- [ ] reviewed "Files changed" and left comments if necessary
+- [ ] confirmed all CI checks have passed
+
+### Reviewers Checklist
+
+*All items are required. Please add a note if the item is not applicable and please add
+your handle next to the items reviewed if you only reviewed selected items.*
+
+I have...
+
+- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
```

Comment:
which reviewer is meant to check these if there are multiple?

Reply:
Good question. I was thinking that each reviewer would be responsible for all checked items unless otherwise noted. If they left their handle next to specific items, it means they did not check all items. If any items are already checked and they have checked all items, then they are indicating that they have checked all items by submitting their review.

In other words, the first reviewer to check the item will check the item. The next reviewer will be responsible for the checked item unless they have left their handle next to specific items.

Reply:
I actually think that specific reviewers may have the role to review specific items - dev rels would review docs and API naming, the architecture/product owner would review API design, a dedicated QA person might review tests, state machine logic and do manual testing, and in general "code owners" (who are not architecture owners) would be responsible for making sure there is a proper review whether they do it themselves or have some people do specific items.

Reply:
> I actually think that specific reviewers may have the role to review specific items

I guess I was thinking this would happen more naturally. I'm not sure how we should implement this and whether it applies to all pull requests but I think this makes sense as something we should work towards.

Does everyone know their role and what items apply to their role? Should this be more clear in the pull request template? Do we need to outline the roles and who fits into what role in the contributing document?